tox.ini

```diff
@@ -55,7 +55,7 @@ commands =
     {env:IGNORE_COVERAGE:} coverage report --fail-under=90 --include *sagemaker_pytorch_serving_container*
 
 deps =
-    coverage
+    coverage==6.2
```

Comment:
Can't we instead update API to coverage 7? I think all you need to do is remove the `--include *sagemaker_pytorch_serving_container*` part from line 55, which isn't really needed anyway given the `--source` argument in line 54.

Reply:
+1

Reply:
@davidthomas426 Done